### DISCUSSION TITLE
conky on sway

### DISCUSSION THREAD

**Discussion by fadri14** [brndnmtthws/conky#1633] [Q&A]
Hello,

I'm on the sway window manager and when I launch conky, it just opens lots of other windows instead of launching on the wallpaper.

Thanks to all who will take the time to help me.

**Comment by MrArborsexual** [comment]
What does your conky config look like?

There are options for outputting to X, Wayland, etc. You may need to look at the reference to pick the right one.

**Comment by fadri14** [comment]
Here's the conky configuration I use:
https://github.com/Kosteron/Conky

I tried to launch conky in scratchpad but it didn't work.

**Reply by MrArborsexual** [reply]
No idea what scratchpad is but try adding this to your conky configs.

out_to_x = false,
out_to_wayland = true,
